src/Types/Resources.ts

```diff
@@ -0,0 +1,27 @@
+export interface Resources {
+  label: string; // Name to be displayed to the user
+  name: string; // unique id for reference
+  description: string; // '' for null
+  tags: string[]; // [] for none
+  dos: string[]; // [] for none
+  donts: string[]; // [] for none
+  liveClip: string; // naration video
+  liveAudio: string; // naration audio handle
```

Comment:
Are these strings because they will be urls to wherever they're stored? Or are they clipIDs who will have their own attributes, like the url?